www/site/content/docs/reference/config.md

```diff
@@ -379,3 +379,27 @@ Default
 
 Used by
 : `chef-run`
+
+### Features
+
+Enable and disable experimental features for Chef Workstation.
+
+#### Example
+
+```toml
+[features]
+preferences_dialog = true
```

Comment:
should this one be added later?

Reply:
I used it as an example, I guess I can ask @kagarmoe to help me with another more generic example? 🤔  (I'm ok removing it or updating it)

Reply:
How does a user discover the possible features?

Reply:
@kagarmoe that is a great question! 💯 I have a proposal that is still under review here: 
https://github.com/chef/chef-workstation-app/pull/125

(it is mentioned in the description of this PR)

Product, UX, and Engineering are still working on a final solution but the idea is to have the
available features listed through the Chef Workstation App, the current design (that I made and won't be the final one) is:
<img width="538" alt="Screen Shot 2019-11-08 at 1 40 26 AM" src="https://user-images.githubusercontent.com/5712253/68816380-899fa680-063b-11ea-9303-8e38a7940b2f.png">

I would like to add that part somewhere in the docs, though, that would probably happen
in approx. two weeks (next week I'm on PTO), for now, this content is what we need until
we have the UI ready.